### PR TITLE
chore: add webcontainer roadmap

### PR DESCRIPTION
**Pull request by Nemikolh**
# PR Description

This PR add a public roadmap for WebContainer

### Self-check

Please check all that apply:

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my code or content update and edited it to the best of my abilities
- [x] I have commented my code, particularly in hard-to-understand areas; my comments are concise

### CODE REVIEW

File: docs/platform/webcontainers/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+The following is intended for informational purposes only and is subject to change. It might contain outdated information or inaccuracies.
+:::
+
+On this page, you'll find information about what is currently being worked on, planned features and things that are under consideration.
+
+If you have a feature request, please check [our github issue tracker](https://github.com/stackblitz/webcontainer-core/issues) to create a new issue or comment to an existing one.
+
+:::info
+This roadmap only includes large features, minor bug fixes or features will not be reflected here
```
Comment (d3lm) [inline docs/platform/webcontainers/roadmap.md:23]: I stumbled when reading this. Adding punctuation would help:
```
This roadmap only includes large features. Minor bug fixes or features will not be reflected here.
```

File: docs/platform/webcontainers/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+:::
+
+## 🚧 In development
```
Comment (d3lm) [inline docs/platform/webcontainers/roadmap.md:26]:
```suggestion
## 🚧 In Development
```

File: docs/platform/webcontainers/roadmap.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+:::info
+This roadmap only includes large features. Minor bug fixes or features will not be reflected here.
+:::
+
+## 🚧 In Development
+
+ * **Node 18** (Github: https://github.com/stackblitz/webcontainer-core/issues/560)
+ * **WASI** (Github: https://github.com/stackblitz/webcontainer-core/issues/705)
+ * **Reducing memory usage** (Github: https://github.com/stackblitz/webcontainer-core/issues/1120)
+ * **NPM Registries** (Github: https://github.com/stackblitz/webcontainer-core/issues/21)
```
Comment (Nemikolh) [inline docs/platform/webcontainers/roadmap.md:31]:
```suggestion
 * **Private / Custom Registries** (Github: https://github.com/stackblitz/webcontainer-core/issues/21)
```

File: docs/platform/webcontainers/roadmap.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+If you have a feature request, please check [our github issue tracker](https://github.com/stackblitz/webcontainer-core/issues) to create a new issue or comment to an existing one.
+
+:::info
+This roadmap only includes large features. Minor bug fixes or features will not be reflected here.
+:::
+
+## 🚧 In Development
+
+ * **Node 18** (Github: https://github.com/stackblitz/webcontainer-core/issues/560)
```
Comment (d3lm) [inline docs/platform/webcontainers/roadmap.md:28]: I think mybe we can add another section `## ✅ Shipped` and we could add `Node 18` to that list.

File: docs/platform/webcontainers/roadmap.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+ * **WASI** (Github: https://github.com/stackblitz/webcontainer-core/issues/705)
```
Comment (d3lm) [inline docs/platform/webcontainers/roadmap.md:29]: Also shipped. We can also link to the blog post.

File: docs/platform/webcontainers/roadmap.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+ * **Reducing memory usage** (Github: https://github.com/stackblitz/webcontainer-core/issues/1120)
+ * **Private / Custom Registries** (Github: https://github.com/stackblitz/webcontainer-core/issues/21)
```
Comment (d3lm) [inline docs/platform/webcontainers/roadmap.md:31]: Shipped and we can link to the teams blog post.

File: docs/platform/webcontainers/roadmap.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+On this page, you'll find information about what is currently being worked on, planned features and things that are under consideration.
+
+If you have a feature request, please check [our github issue tracker](https://github.com/stackblitz/webcontainer-core/issues) to create a new issue or comment to an existing one.
+
+:::info
+This roadmap only includes large features. Minor bug fixes or features will not be reflected here.
+:::
+
+## 🚧 In Development
```
Comment (d3lm) [inline docs/platform/webcontainers/roadmap.md:26]: What we could list as well:
- CORS Proxy

**Review by d3lm — COMMENTED** [review]
Left a few minor remarks. Otherwise this looks good 👍

File: docs/platform/webcontainers/roadmap.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+ * **Node 18** (Github: https://github.com/stackblitz/webcontainer-core/issues/560)
+ * **WASI** (Github: https://github.com/stackblitz/webcontainer-core/issues/705)
+ * **Reducing memory usage** (Github: https://github.com/stackblitz/webcontainer-core/issues/1120)
+ * **Private / Custom Registries** (Github: https://github.com/stackblitz/webcontainer-core/issues/21)
+
+## 🦄 Future
```
Comment (d3lm) [inline docs/platform/webcontainers/roadmap.md:33]: We can also mention `Supporting multiple Node.js versions`. I think it's something we can and should commit to.

**Review by d3lm — APPROVED** [review]

**Pull request merged**